charts/alluxio/templates/worker/statefulset.yaml

```diff
@@ -15,7 +15,7 @@
 {{- $hostPID := .Values.worker.hostPID }}
 
 apiVersion: apps/v1
-kind: DaemonSet
+kind: StatefulSet
```

Comment:
Could you also update the `Chart.yaml` and `CHANGELOG.md` ?

Reply:
thanks, Chart.yaml and CHANGELOG.md updated